### DISCUSSION TITLE
Error Occurred : '异常'

### DISCUSSION THREAD

**Discussion by YE-JR** [PaddlePaddle/PaddleSpatial#114] [BaiduKDDCup2022] [answered]
Dear Organizer,
I got an error ' Error Occurred : '异常' ' My codebase can pass local test. What is the meaning of this error?
My team name is 'JRTeam'
![image](https://user-images.githubusercontent.com/84713815/170696373-3a0e453b-de29-4532-8a37-b6b45ca9077d.png)

**Comment by ramber1836** [comment, marked as answer]
The error is: 
![image](https://user-images.githubusercontent.com/19561583/170717578-d84825a6-4800-475d-bf9f-1f34efe2f303.png)

You cannot use the argument parser in your own code as it will conflict with the online test script eval_main.py
Please refer to the manual (PaddleSpatial/apps/wpf_baseline_gru/kddcup22-sdwpf-evaluation/README.md) for more details on how to submit your code.

**Comment by YE-JR** [comment]
Thank you for your reply！


------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "PaddlePaddle/PaddleSpatial"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年5月27日(星期五) 晚上10:23
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [PaddlePaddle/PaddleSpatial] Error Occurred : '异常' (Discussion #114)





 
The error is:
 
 
You cannot use the argument parser in your own code as it will conflict with the online test script eval_main.py
 Please refer to the manual (PaddleSpatial/apps/wpf_baseline_gru/kddcup22-sdwpf-evaluation/README.md) for more details on how to submit your code.
